pkg/sql/lease.go

```diff
@@ -1507,7 +1510,7 @@ func NewLeaseManager(
 	return lm
 }
 
-func nameMatchesTable(
+func NameMatchesTable(
```

Comment:
12th commit: this function needs a comment

Reply:
Done.